webpack-and-babel/imports.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+
+// import from a file located at the absolute path `/foo/bar/baz/qux.js`:
+// eslint-disable-next-line import/no-absolute-path
+import f1 from '/foo/bar/baz/qux.js';
```

Comment:
You don't need the `.js`. Webpack will try and add file extensions automatically for `.js`, `.jsx`, `.ts`, `.tsx`